README.md

```diff
@@ -4,6 +4,7 @@
 * [Coding Style and Standards](coding_style_and_standards.md)
 * [Developer Setup](developer_setup.md)
 * [Development Appliance Setup](https://github.com/ManageIQ/manageiq-appliance-dev-setup)
+* [Developer copr setup](https://github.com/ManageIQ/developer_copr_setup.md)
```

Comment:
I don't think so, https://copr.fedoraproject.org/coprs/, it's only the first character that's capitalized.